src/authorization.ts

```diff
@@ -5,7 +5,8 @@ const authContextHasFunderPermission = (
 	funderShortCode: ShortCode,
 	permission: Permission,
 ): boolean =>
-	auth.user.permissions.funder[funderShortCode] !== undefined &&
-	auth.user.permissions.funder[funderShortCode].includes(permission);
+	auth.role.isAdministrator ||
```

Comment:
I believe this is the right place for this check (rather than having an explicit check every time a funder permission is tested).

If we ever have a case where we don't want the administrator flag to grant permission for some reason then I think that is a significant enough departure from the intention of `superuser` that a refactor would be appropriate at that point.